src/demo/configs.js

```diff
@@ -48,7 +48,7 @@ export const configs = {
   'ome-ngff-v0.1': omeNgffLegacy,
   // Keys which enable backwards compatibility with old links.
   'linnarsson-2018': notPublic(codeluppi2018),
-  gating: codeluppiGating,
+  gating: notPublic(codeluppiGating),
```

Comment:
this doesn't seem to be working? or is this only for the docs site?

Reply:
I made this change after deploying the demos so that is expected when you click the demo link, but it seems to be working locally